src/main/java/com/gooddata/authentication/LoginPasswordAuthentication.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2004-2016, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.authentication;
+
+import com.gooddata.GoodDataEndpoint;
+import com.gooddata.Authentication;
+import com.gooddata.http.client.GoodDataHttpClient;
+import com.gooddata.http.client.LoginSSTRetrievalStrategy;
+import com.gooddata.http.client.SSTRetrievalStrategy;
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import static com.gooddata.util.Validate.notEmpty;
+import static com.gooddata.util.Validate.notNull;
+
+/**
+ * This class provides SST authentication (using login and password).
```

Comment:
please try to impersonate a user who know anything about internal implementation of gooddata authentication and who has no idea what SST means.

"Login and password authentication" should be enough